packages/language/system-test/language.js

```diff
@@ -339,66 +310,20 @@ describe('Language', function() {
     });
   }
 
-  function validateAnnotationSimple(callback) {
-    return function(err, annotation, apiResponse) {
-      try {
-        assert.ifError(err);
-
-        assert.strictEqual(annotation.language, 'en');
-
-        assert(is.number(annotation.sentiment));
-
-        assert.deepEqual(annotation.entities, {
-          people: ['stephen', 'david'],
-          places: ['michigan']
-        });
-
-        assert.deepEqual(annotation.sentences, TEXT_CONTENT_SENTENCES);
-
-        assert(is.array(annotation.tokens));
-        assert.deepEqual(annotation.tokens[0], {
-          text: 'Hello',
-          partOfSpeech: 'Other: foreign words, typos, abbreviations',
-          tag: 'X',
-          aspect: undefined,
-          case: undefined,
-          form: undefined,
-          gender: undefined,
-          mood: undefined,
-          number: undefined,
-          person: undefined,
-          proper: undefined,
-          reciprocity: undefined,
-          tense: undefined,
-          voice: undefined,
-          dependencyEdge: {
-            description: 'Root',
-            label: 'ROOT',
-            headTokenIndex: 0
-          }
-        });
-
-        assert(is.object(apiResponse));
-
-        callback();
-      } catch(e) {
-        callback(e);
-      }
-    };
-  }
-
-  function validateAnnotationVerbose(callback) {
+  function validateAnnotation(callback) {
     return function(err, annotation, apiResponse) {
       try {
         assert.ifError(err);
 
         assert.strictEqual(annotation.language, 'en');
 
         assert(is.object(annotation.sentiment));
+        assert(is.number(annotation.sentiment.score));
```

Comment:
For these, let's just check that `annotation` === apiResponse.sentiment` (or whatever). Keep anything that checks that the correct amount of results are returned, but type-checking isn't necessary anymore.